### PR TITLE
Adjust desktop menu

### PR DESCRIPTION
**Pull request by CalebCyphers**
#### What's this PR do?

This PR builds on stacy's styling for the desktop view of the menu

#### Where should the reviewer start?

styles.css. line 396 (media queries)

#### How should this be manually tested?

- open index.html
- adjust sizing to test breakpoints in dev tools
- add and remove the new 'hidden' class from the 'open-nav-filter' on line 9 in index.html

### CODE REVIEW

**Review by stacyp2006 — APPROVED** [review]
Great work!

File: styles.css [hunk 1]
```diff
@@ -398,7 +413,10 @@ body {
   .header-text {
     margin-top: 0.5rem;
     justify-self: flex-start;
-    border-bottom: 1px solid #353567;
+  }
+
+  .header-text-wrapper {
+    width: 100%
```
Comment (MichaelEWalker87) [inline styles.css:419]: missing a semi ;

File: styles.css [hunk 2]
```diff
@@ -12,6 +12,7 @@ html, body {
 body {
   display: flex;
   flex-direction: column;
+  height: 100%
```
Comment (MichaelEWalker87) [inline styles.css:15]: missing a semi colon

**Review by MichaelEWalker87 — APPROVED** [review]
I might look at adjusting the media query and adding a tablet

**Pull request merged**